packages/containers/src/List/List.connect.js

```diff
@@ -6,29 +6,40 @@ import {
 	configureGetFilteredItems,
 	configureGetPagination,
 	configureGetPagedItems,
+	getCollectionItems,
+	configureGetSortedItems,
 } from './selector';
 
 function componentId(ownProps) {
 	return ownProps.collectionId;
 }
 
 function getItems(state, config) {
-	const items = configureGetFilteredItems(config)(state);
-
-	return items || new List();
+	let items = config.items;
+	if (config.defaultFilter !== false) {
+		items = configureGetFilteredItems(config)(state);
+	}
+	if (config.defaultSorting !== false) {
+		items = configureGetSortedItems(config, items)(state);
+	}
+	return items;
 }
 
-function getPagedItems(state, config) {
-	const items = configureGetPagedItems(config)(state);
-
-	return items || new List();
+function getPagedItems(state, config, items) {
+	if (config.defaultFilter !== false && config.defaultSorting !== false) {
+		return configureGetPagedItems(config, items)(state);
+	}
+	return config.items;
```

Comment:
Isn't that `items` you should return ?

Reply:
this function returns the paged items except when the defaultPaging is set to false

Reply:
and yes the function should return the items, its fixed now